extensions/wikia/UserPreferencesV2/UserPreferencesV2.i18n.php

```diff
@@ -47,10 +46,9 @@
 	'preferences-v2-watchdeletion' => 'Delete',
 	'preferences-v2-watchcreations' => 'Create',
 	'preferences-v2-showads-enable' => 'Show all ads',
-	'preferences-v2-showads-disable' => 'Don\'t show all ads',
+	'preferences-v2-showads-disable' => "Don't show all ads",
 	'preferences-v2-my-email-address' => 'My email address:',
 	'prefs-patrolled-editsv2' => 'Patrolled edits',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "prefs-patrolled-editsv2" 28 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)